packages/manager/apps/dedicated/client/app/ip/components/list/byoip/aggregate/ip-ip-byoip-aggregate.controller.js

```diff
@@ -0,0 +1,43 @@
+export default ($scope, IpByoipService) => {
```

Comment:
Can you define controllers with classes please.
Also define component with a proper component file (defining bindings and template) and declare it in the module using angularjs's component method (you can take example of the stepper which is declared inside the packages/manager/apps/dedicated/client/app/ip/components/ folder)

Reply:
I changed to a class but i can't use a component because it's used as a template in the setAction in ipBlock
If you have an example of component used with setAction you can show me.